packages/backend/src/tokens/request.ts

```diff
@@ -17,15 +17,15 @@ import type { AuthenticateRequestOptions } from './types';
 import { verifyToken } from './verify';
 
 const RefreshTokenErrorReason = {
-  NoCookie: 'no-cookie',
-  NonEligible: 'non-eligible',
+  NonEligibleNoCookie: 'non-eligible-no-refresh-cookie',
+  NonEligibleNonGet: 'non-eligible-non-get',
   InvalidSessionToken: 'invalid-session-token',
   MissingApiClient: 'missing-api-client',
   MissingSessionToken: 'missing-session-token',
   MissingRefreshToken: 'missing-refresh-token',
```

Comment:
❓ What's the difference between this and `session-token-expired-refresh-non-eligible-no-refresh-cookie`?

Reply:
These values are not used alone. They get concatenated with the `session-token-expired-refresh-` prefix. So, it will result in the one you specified.